build/mvn

```diff
@@ -69,14 +69,23 @@ install_app() {
 
 # Install maven under the build/ folder
 install_mvn() {
-  local APACHE_MIRROR=${APACHE_MIRROR:-'https://www.apache.org/dyn/closer.lua?action=download&filename='}
+  MVN_VERSION="3.3.9"
```

Comment:
I mean that using this build/mvn script should work even if there isn't any version of maven currently installed on the machine (which might be the case, e.g., when using an automated tool to set up a new build node.)  I believe that would work with `--force` previously, and it should work when there isn't a `--force` option anymore.